### DISCUSSION TITLE
Activation failed

### DISCUSSION THREAD

**Discussion by PathumB** [community/community#113549] [Copilot Conversations]
### Select Topic Area

General

### Body

2024-03-21 14:01:45.226 [info] [auth] Invalid copilot token: missing token: 403 
2024-03-21 14:01:45.362 [error] [default] Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as PathumB."

Previously i have activate this by university student ID. Now it expired. But still i'm a university student. please help me to get this.

**Comment by Akash1134** [comment]
Hi @PathumB ,

It seems you need to re-verify your academic affiliation. We periodically require previously-verified participants to re-confirm their current academic status.
 
Please visit [GitHub Education](https://education.github.com/discount_requests/application) to submit a new request to verify your academic status.
 
Once your academic status is successfully verified you will gain immediate access to GitHub Global Campus. Additional academic benefits will become available within 72 hours of your successful verification.

Thanks!
